lerna.json

```diff
@@ -6,6 +6,9 @@
   "command": {
     "run": {
       "stream": true
+    },
+    "publish": {
+      "skipGit": true
```

Comment:
For my own information, what is `skipGit` used for and why was it necessary as a change?
Perhaps this is something useful to include as part of the commit body as it doesn't look trivial in relation to the PR.

Reply:
As discuss with Marc, we should be able to look at the modified files before committing to git. With this flag, when running the publish command, it does not create automatically a commit and push.
This is the same behavior as the main theia branch.